tools/rapids-conda-retry

```diff
@@ -151,16 +168,17 @@ function runConda {
 segfault exit code 139"
         fi
 
-        if (( needToRetry == 1 )) && \
-           (( retries >= max_retries )); then
-            # Catch instance where we run out of retries
-            rapids-echo-stderr "Exiting, reached max retries..."
-        else
-            # Give reason for retry
-            rapids-echo-stderr "${retryingMsg}"
-            if (( needToClean == 1 )); then
-                rapids-echo-stderr "Cleaning tarball cache before retrying..."
-                ${condaCmd} clean --tarballs -y
+        if (( needToRetry == 1 )); then
```

Comment:
Re-formatting this because in the previous example, `rapids-echo-stderr "${retryingMsg}"` always ran, resulting in 1 empty log line in the case where `needToRetry` is not `1`.